### PR TITLE
chore: document the stack.tags attribute.

### PR DESCRIPTION
**Pull request by i4ki**
# Reason for This Change

Document the `stack.tags` attribute.

### CODE REVIEW

**Review by soerenmartius — APPROVED** [review]
lgtm

**Review by mariux — APPROVED** [review]
lgtm

File: docs/sharing-data.md [hunk 1]
```diff
@@ -418,6 +418,13 @@ The default value is an empty string.
 Please consider [stack configuration](stack.md) to see how
 you can change the default stack description.
 
+### terramate.stack.tags (list)
+
+The list of stack tags. The default value is an empty list.
+
+Please consider [stack configuration](stack.md) to see how
+you can change the stack tags.
```
Comment (mariux) [inline docs/sharing-data.md:426]:
```suggestion
Please consider [stack configuration](stack.md) to see how you can change the stack tags.
```

File: docs/stack.md [hunk 2]
```diff
@@ -65,6 +65,15 @@ stack {
 }
 ```
 
+## stack.tags (set(string))(optional)
+
+The tags must be a unique set of strings, where each tag must adhere to the
+following rules:
```
Comment (mariux) [inline docs/stack.md:71]:
```suggestion
The tags must be a unique set of strings, where each tag must adhere to the following rules:
```

**Review by mariux — COMMENTED** [review]
added comments to let the renderer decide where to break lines.

**Review by switchdk — APPROVED** [review]
LGTM :+1:

**Pull request merged**